src/sinks/blackhole.rs

```diff
@@ -71,7 +72,8 @@ impl Sink for BlackholeSink {
         self.total_events += 1;
         self.total_raw_bytes += message_len;
 
-        trace!(raw_bytes_counter = message_len, events_counter = 1);
+        counter!("blackhole.events", 1);
```

Comment:
idea: may be nice to start doing

```rust
tracing::info!
metrics::counter!
```

This could just make the context really clear in the location. Just an idea, may be too tedious in practice.

Reply:
Also keeps imports simple!